### PR TITLE
add relatedInfo function

### LINKED ISSUE #32: Add related info to all 3 stories
## Description
Refactor story deck in contemplating action to allow related info to be added by brands

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #32 

## Description

- Added relatedInfo function

@neontribe/contemplating-action

🐧

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -542,3 +542,19 @@ brandCopy key =
...
+
+
+relatedInfo : Int -> List Int
+relatedInfo storyId =
+    case storyId of
+        1 ->
+            [ 1, 2 ]
+
+        2 ->
+            [ 2, 3 ]
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Did you get from @roseb04 or shall we go with 1,2 3,4 5,6 so they all get exposure?

Comment (katjam) [reply src/Copy/BrandCopy.elm]: We could also read the stories and pick relevant ones ourselves. :)

**Comment by rosejbon** [comment]
I've put the associated information in the cards with the stories, sorry
should have moved them out and put in relevant ticket. They can be found in
he acceptance criteria

**Comment by katjam** [comment]
No probs. I thought I'd seen somewhere. We were being blind.

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -542,3 +542,19 @@ brandCopy key =
 
         StoryThree4ImageAlt ->
             CopyText "Illustration of a man looking relaxed and happy in a garden. He is in deep conversation with a woman who is smiling."
+
+
+relatedInfo : Int -> List Int
+relatedInfo storyId =
+    case storyId of
+        1 ->
+            [ 1, 2 ]
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: As per https://github.com/neontribe/ca-hospice-hope/issues/5 acceptance criteria should be 2,4

File: src/Copy/BrandCopy.elm (hunk 1 above)
Comment (katjam) [inline src/Copy/BrandCopy.elm]: As per https://github.com/neontribe/ca-hospice-hope/issues/3 acceptance should be 1,3

File: src/Copy/BrandCopy.elm [hunk 3]
```diff
@@ -542,3 +542,19 @@ brandCopy key =
...
+
+        2 ->
+            [ 2, 3 ]
+
+        3 ->
+            [ 3, 4 ]
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: As per https://github.com/neontribe/ca-hospice-hope/issues/4 acceptance should be 5,6

**Review by katjam — CHANGES_REQUESTED** [review]
Found the related info id's (with help from @rose)

**Review by katjam — APPROVED** [review]
Noice :lollipop:

**Pull request merged**